CORPSE_deriv.py

```diff
@@ -136,24 +138,32 @@ def CORPSE_deriv(SOM,T,theta,Ndemand,Ctransfer,params,claymod=1.0):
         # C and N available for microbial growth
         if mt=='SAP':
            for t in chem_types:
-               carbon_supply[mt]=carbon_supply[mt]+decomp[t+'C']*params['eup'][t]
+               if t=='Slow':
+                   CUE_SAP = 0.61 - 0.012 * (T - 273)  # From DeVêvre and Horwáth (2000)
+               else:
+                   CUE_SAP = 0.3512 - 0.0095 * (T - 273)  # Frey et al. (2013) (Phenols)
+               carbon_supply[mt]=carbon_supply[mt]+decomp[t+'C']*CUE_SAP
                nitrogen_supply[mt]=nitrogen_supply[mt]+decomp[t+'N']*params['nup'][t]
-               IMM_N_max = T_factor(T,params) * params['max_scavenging_rate']['SAP'] * SOM['inorganicN'] / (
+               IMM_N_max = T_factor(T,params,'InorgN') * params['max_scavenging_rate']['SAP'] * SOM['inorganicN'] / (
                        SOM['inorganicN'] + params['kc_scavenging_IN']['SAP'] * params['depth']) \
                                  * SOM['SAPC'] / (SOM['SAPC'] + params['kc_scavenging']['SAP'] * params['depth'])
         else:
-           carbon_supply[mt] += Ctransfer[mt]*params['eup_myc'][mt]
+           carbon_supply[mt] += SOM['Int_'+mt+'C'] * params['eup_myc'][mt]
```

Comment:
I would add a parameter for the time scale of transfer from the intermediate pool into mycorrhizal biomass. Currently, `SOM['Int_ECMC']` (for example) is a pool (units of g C) and `carbon_supply` is a rate ( g C/year). So this calculation will give the intermediate pool a fixed time scale of 1.0 year<sup>-1</sup>. That is, if there is 1 g of C in the intermediate pool then the instantaneous C supply from that pool equals 1 g/year (multiplied by CUE). If you add a rate parameter (units of year<sup>-1</sup>) then you could make that rate faster which I think would be more realistic.